bin/inx-chronicle/src/api/error.rs

```diff
@@ -41,19 +41,25 @@ pub enum ApiError {
     BadParse(ParseError),
     #[error("Invalid time range")]
     BadTimeRange,
+    // TODO: use or remove dead code
```

Comment:
Do you plan on doing these `TODO`s in this PR?

Reply:
I'm not sure because I didnt introduce those variants myself. Would have to talk to @AlexandCoats about it first, and see if they are needed at some point.

Reply:
These were part of the indexation API, so they can be removed.